### ISSUE TITLE
Deprecation Announcement

### ISSUE THREAD

**Issue by EHfive** [EHfive/pulseaudio-modules-bt#154]
Since I have already switched to PipeWire and I don't want to pay further efforts on this. I am going to deprecate this project. The repo will be archived on January 25.

Although the modules are still working on current PulseAudio Releases(12.0 - 14.1) and later API compatible releases. I would recommend you to switch to vanilla PulseAudio or PipeWire. 
There is a WIP MR adding extra A2DP codecs support on PulseAudio https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/440, PulseAudio users may count on this.

**Comment by soredake** [comment]
I hope ppa will be updated for 21.04, new pulseaudio version is not released, so 21.04 will have vanilla pulseaudio without codecs.

**Comment by soredake** [comment]
@berglh will you update ppa to 21.04 for the last time?

**Comment by bcremer** [comment]
Thanks for all the work you put in this on the last 2+ years.

**Comment by berglh** [comment]
> @berglh will you update ppa to 21.04 for the last time?

@soredake Absolutely. I had a feeling that LDAC wasn't being included in the original MR for PulseAudio, but it now looks like the MR has a lease of new life. We'll keep an eye on this MR, if it's landed and functional for 21.04, I won't bother though.

**Comment by lucasfijen** [comment]
@EHfive and @berglh Thank you so much for your effort on this repo! I've been enjoying ldac so much!

Glad it'll now be implemented in a more official way so that more people will be able to enjoy good audio:)

**Comment by soredake** [comment]
> if it's landed and functional for 21.04

Feature freeze will happen [soon](https://discourse.ubuntu.com/t/hirsute-hippo-release-schedule/18539) :(

**Comment by berglh** [comment]
@EHfive Thanks for all your personal help and guidance for bringing your excellent work to the Ubuntu users.
@soredake I will build closer to the release date, open a new issue and tag me in it when it's launch day. I'll upgrade my laptop and get it tested and built.

**Comment by artizirk** [comment]
PulseAudio codecs support was just merged upstream! https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/440

**Comment by soredake** [comment]
https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/476

**Comment by Roger-ZRZ** [comment]
Sorry to ask, but are there any other stable alternatives we can use to stream AptX or LDAC for now?

**Comment by EHfive** [comment]
> Sorry to ask, but are there any other stable alternatives we can use to stream AptX or LDAC for now? 

PipeWire.

https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/NEWS

**Comment by IDeathByte** [comment]
PipeWire supports extended SBC options now?
